src/internet_identity/src/main.rs

```diff
@@ -233,14 +219,15 @@ fn post_upgrade(maybe_arg: Option<InternetIdentityInit>) {
     // load the persistent state after initializing storage, otherwise the memory address to load it from cannot be calculated
     state::load_persistent_state();
 
+    apply_init_arg(maybe_arg);
+}
+
+fn apply_init_arg(maybe_arg: Option<InternetIdentityInit>) {
```

Comment:
small nit about the name, it's not really "init" arg now is it?

Reply:
Renamed to `apply_install_arg` 👍